dev/io.openliberty.jakarta.cdi.4.0/bnd.bnd

```diff
@@ -11,38 +11,44 @@
 -include= ~../cnf/resources/bnd/bundle.props
 bVersion=1.0
 
```

Comment:
Should have:
```
javac.source: 11
javac.target: 11

Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
```
added to the bnd file and then update the .classpath and org.eclipse.jdt.core.prefs to have 11 instead of 1.8 in its java settings.